src/components/Navbar/Navbar.tsx

```diff
@@ -2,8 +2,9 @@ import React, { useState } from 'react'
 import './Navbar.css'
 import { NavLink, useLocation } from 'react-router-dom'
 import { RxHamburgerMenu, RxCross2 } from "react-icons/rx"
+import { User } from '../../Interfaces'
 
-const Navbar = () => {
+const Navbar = ({currentUser}: {currentUser: User | any}) => {
```

Comment:
Per my comments above, this wouldn't be necessary. But probably really good practice in react in passing props around. Right underneath this line is where we can access local storage and save it to state using the following lines of code: 
`const currentUser : any = window.localStorage.getItem("currentUser")`
`const [ user, setUser ] = useState(JSON.parse(currentUser))`